static/app/components/modals/inviteMembersModal/inviteRowControl.tsx

```diff
@@ -79,8 +79,9 @@ function InviteRowControl({
   };
 
   return (
-    <div className={className}>
+    <li className={className}>
```

Comment:
maybe we can replace this with our list component?

Reply:
nah, our list component has styling, I just wanted this to be a list

Reply:
doesn't it?

Reply:
yes it has some padding left